### PR TITLE
Added Updated gif and email ID

### PR DESCRIPTION
**Pull request by SrijanJha28**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,9 +108,11 @@ Or use the 1-click deployment button
 
  - Once deployed, you'll find the generator running on http://localhost:5000, it should look like this  
 
-![Generator Screencast](docs/screenshots/screencast.gif)
+![Generator Screencast](docs/screenshots/openevent1.gif)
 
- - Add your **email id**, **name of app** (name of event),
+srijanjha09@gmail.com
```
Comment (the-dagger) [inline README.md]: It is not required for you to actually enter your email here.
If you read the Readme carefully, these are the steps which guide the user on how to use this generator.

**Comment by the-dagger** [comment]
Also the gif that you have uploaded is not really necessary as the readme already contains a gif in abetter resolution.

**Comment by SrijanJha28** [comment]
But princu7 told me update the gif and write the email id

**Comment by the-dagger** [comment]
Nop, you don't have to do that.
There are some other tasks related to the Open Event Webapp that are up for grab, so you can have a look at them.
Or better yet, try and fix any errors (grammatical or structural) in the Readme. 🙂

**Comment by Princu7** [comment]
@SrijanJha28 I guess it was misunderstanding. When I said write email first, I meant that in the context of gif. That is, in the initial gif, he was first uploading the zip file and then writing his email address in the form. I wanted you to follow the chronological order and enter the email address first and then upload the zip. :smile: Writing email in Readme is certainly wrong.

**Comment by Princu7** [comment]
@SrijanJha28 Please improve the quality of the gif as stated by @the-dagger. Also, the gif is too short. It would be better if you showed the actual completion of the build process and a preview of the generated website. You can use EHealth Africa Summit zip file for this purpose since it is short and won't take much time to generate. Thanks.

**Comment by aayusharora** [comment]
@the-dagger @Princu7 Thanks! @SrijanJha28 Which issue is it related to ?

**Comment by SrijanJha28** [comment]
I made the video but princu7 told me to add updated gif and not the video

**Comment by Princu7** [comment]
@aayusharora The existing gif was a bit old. I thought it would be better to have an updated one. Since the video would demonstrate almost the same process as the gif, I thought in favor of the gif.  Please suggest. :+1:

**Comment by SrijanJha28** [comment]
Online converting video to gif lowers the standard of resolution

**Comment by aayusharora** [comment]
Ok! @Princu7 @SrijanJha28  an updated gif will be better option, if it gives the required process flow. Most of the times it seems heavy which increases loading time of readme. Let's try to update it with a compressed version with only needed information.
Kindly take further discussions related to this PR to #924.

**Comment by mariobehling** [comment]
@SrijanJha28 Still up for it? Would be great to see your gif getting included.

**Comment by mariobehling** [comment]
@SrijanJha28 I am closing this PR due to inactivity. Please re-commit if you would like to get back working on it. Cheers.

**Pull request closed without merging**